src/test/controller/BaseController.ts

```diff
@@ -5,71 +5,97 @@ import { auth } from 'firebase-admin/lib/auth';
 import Auth = auth.Auth;
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const firebaseMock = require('firebase-mock');
-import { AxiosInstance } from 'axios';
+import axios, { AxiosInstance } from 'axios';
 import nodeGeocoder, { Geocoder } from 'node-geocoder';
+import AWSMock from 'mock-aws-s3';
+import { S3 } from 'aws-sdk';
+import { S3_BUCKET } from '../../main/service/ResourceCleanupService';
```

Comment:
I can't remember if we have other constants like this, but maybe potentially adding a config file if we have more?

Reply:
I think it should be okay since this is one constant and its in a service class. It's usage is only in the service class asides from test classes using it